backends/server/package.json

```diff
@@ -5,11 +5,21 @@
   "main": "server.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "node server.js"
+    "build": "yarn tsc",
+    "prestart": "yarn build",
+    "start": "node dist/server.js",
+    "preserve": "yarn build",
```

Comment:
I LOVED the addition of pre-scripts!
I think we don't need the preserve because you already doing "yarn tsc -w" on the serve command